swaps/rebalance.js

```diff
@@ -596,6 +598,8 @@ module.exports = (args, cbk) => {
           return cbk([400, 'ExpectedDifferentPeersForInboundAndOutbound']);
         }
 
+        const maxFee = !!args.max_fee_rate ? round((args.max_fee_rate * max) / 1e6) : round((defaultMaxFeeRate * max) / 1e6);
```

Comment:
i think this should be based on tokens rather than max, and it should be based on mtokens

Reply:
Fixed, there is a mtokens conversion before executing the probe.